istioctl/cmd/proxyconfig.go

```diff
@@ -76,13 +76,13 @@ const (
 // existing sorted active loggers
 var activeLoggers = []string{
 	"admin",
-	"all",
```

Comment:
just to clarify, we don't need `all` anymore since I can do `--level trace` instead?

Reply:
I tested `--level <level>` and it worked.  I only tested with the Envoy that was on some 1.4 alpha sidecars I was running.  It is possible this wouldn't work on an older Envoy, although I suspect it will.

I don't know where `all` originally come from, or which versions of Envoy supported it.  I remember setting the level using `curl` without `all`.

Reply:
FYI, https://istio.io/docs/reference/commands/istioctl/#istioctl-proxy-config-log still references the "all" parameter.